lib/offsite_payments/integrations/realex_offsite.rb

```diff
@@ -424,10 +424,6 @@ def message
           params['MESSAGE']
         end
 
-        def pasref
```

Comment:
This is used too, here https://github.com/activemerchant/offsite_payments/blob/e71fef436de17b0bc18a5bfa7ddeeef69a399fce/lib/offsite_payments/integrations/realex_offsite.rb#L372

Reply:
nvm, they're defined twice in the file :this_is_fine: